utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -1156,6 +1156,11 @@ inline void compare_output(T* output, string funcName, RpptDescPtr srcDescPtr, R
         func += "_noiseType" + noiseTypeName;
         binFile += "_noiseType" + noiseTypeName;
     }
+    else if(testCase == 54)
+    {
+        func += "_kSize" + std::to_string(additionalParam);
+        binFile += "_kSize" + std::to_string(additionalParam);
```

Comment:
Please change this to _kernelSize to maintain consistency

Reply:
Done